tools/scripts/ath10k/ath10k-bdencoder

```diff
@@ -600,6 +603,11 @@ def git_get_head_id():
 
 
 def cmd_add_mbox(args):
+    if args.mode == 10:
+        firmware_url = ATH10K_FIRMWARE_URL
+    elif args.mode == 1:
+        firmware_url = ATH10K_FIRMWARE_URL
```

Comment:
```diff
+elif args.mode == 11:
 +       firmware_url = ATH11K_FIRMWARE_URL
```

I suspect that it might be better make mode a global variable and add getters using the mode to return signature/firmware url

Reply:
Thank you!